.gitignore

```diff
@@ -4,3 +4,4 @@
 # Helm
 **/charts/*.tgz
 helm-charts/splunk-otel-collector/Chart.lock
+.DS_Store
```

Comment:
nit: make this a separate PR? I ended up myself doing this:
```
> cat ~/.gitignore_global
.idea
.DS_Store
```

Reply:
I deleted .DS_Store entry from .gitignore